erb_lint.gemspec

```diff
@@ -1,21 +1,28 @@
 # frozen_string_literal: true
 
+$LOAD_PATH.push File.expand_path("../lib", __FILE__)
+
+require 'erb_lint/version'
+
 Gem::Specification.new do |s|
   s.name = 'erb_lint'
-  s.version = '0.0.9'
+  s.version = ERBLint::VERSION
   s.authors = ['Justin Chan']
   s.email = ['justin.the.c@gmail.com']
   s.summary = 'ERB lint tool'
   s.description = 'ERB Linter tool.'
   s.homepage = 'https://github.com/justinthec/erb-lint'
   s.license = 'MIT'
 
-  s.files = Dir['lib/**/*.rb']
+  s.files = Dir['lib/**/*.rb', 'bin/*']
+  s.bindir = 'exe'
+  s.executables = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
```

Comment:
Is this OK with both of these?

Reply:
We should be using `s.files.grep(%r{^exe/})`.

Reply:
it's fixed now